pages/members/index.tsx

```diff
@@ -72,26 +79,40 @@ const HonoraryMembersPage: NextPage = () => {
           justifyContent="center"
           className={styles.buttonStyle}
         >
-          <Button
-            size="large"
-            variant="contained"
-            color="error"
-            onClick={() => {
+          <ToggleButton
+            value="check"
+            selected={honorSelected}
+            onChange={async () => {
+              if (!honorSelected && orderSlected) {
+                setOrderSlected(!orderSlected);
+                setHonorSelected(!honorSelected);
+              }
               setPageDisplay(true);
+              router.query.view = "honorary-members";
+              await router.push(router);
```

Comment:
I think `router.replace` would be better here, since it's sort of not a "new" page in that sense to the user. So we just keep a single item in the history when navigating between the order and honarary member pages.

Reply:
But this is just up to preference.